systemd-swap

```diff
@@ -642,7 +642,8 @@ def stop(on_init: bool = False) -> None:
     sem.remove()
 
 def status() -> None:
-    init()
+    am_i_root()
```

Comment:
This is totally unneeded, we don't actually need root for the status sub command (right?)

Reply:
Pff ignore me

Reply:
I do however think, from a design perspective or w/e that the status should be accessible to non root users aswell

Reply:
Implemented. Only Zswap stats should be unavailable as non-root user.

Reply:
Oh thank you ^^, your doing all the work for me

Reply:
You're welcome. It was only a small change and some testing.